app/javascript/elm/src/Main.elm

```diff
@@ -922,6 +917,27 @@ viewSensorFilter sensors selectedSensorId =
         ]
 
 
+viewCrowdMapOptions : Bool -> Int -> Bool -> Html Msg
+viewCrowdMapOptions isCrowdMapOn crowdMapResolution isDisabled =
+    let
+        class_ =
+            case isDisabled of
+                True ->
+                    "disabled-area"
+
+                False ->
+                    ""
```

Comment:
Is there a better/shorter way to have a class dependent on some bool value?

Reply:
Yep :) https://package.elm-lang.org/packages/elm/html/latest/Html-Attributes#classList